chia/_tests/core/full_node/test_full_node.py

```diff
@@ -2683,7 +2683,7 @@ def check_nodes_in_sync() -> bool:
         p2 = full_node_1.full_node.blockchain.get_peak()
         return p1 == p2
 
-    await time_out_assert(100, check_nodes_in_sync)
+    await time_out_assert(200, check_nodes_in_sync)
```

Comment:
any idea why this was needed ?

Reply:
I believe there are more transactions per block this way. I'm not entirely sure why there would be more, but the chains were slightly larger on disk